incubator/hnc/pkg/stats/stats.go

```diff
@@ -76,39 +64,28 @@ func StartObjReconcile(gvk schema.GroupVersionKind) {
 // StopObjReconcile updates the stats for objects with common GK
 // when an object reconciliation finishes.
 func StopObjReconcile(gvk schema.GroupVersionKind) {
-	if !testLogEnabled {
-		return
-	}
 	gk := gvk.GroupKind()
 	stats.objects[gk].curReconciles.decr()
 }
 
 // WriteNamespace updates stats when writing namespace instance.
 func WriteNamespace() {
-	if !testLogEnabled {
-		return
-	}
 	stats.namespaceWrites.incr()
 }
 
 // WriteHierConfig updates stats when writing hierarchyConfig instance.
 func WriteHierConfig() {
-	if !testLogEnabled {
-		return
-	}
 	stats.hierConfigWrites.incr()
 }
 
 // WriteObject updates the object stats by GK when writing the object.
 func WriteObject(gvk schema.GroupVersionKind) {
-	if !testLogEnabled {
-		return
-	}
 	gk := gvk.GroupKind()
 	stats.objects[gk].apiWrites.incr()
 }
 
-func initStats() {
+// Init initializes the stat counters.
+func Init() {
```

Comment:
If you just call this `init()`, it will be called when HNC starts and you don't need to call it manually (e.g. see https://medium.com/golangspec/init-functions-in-go-eac191b3860a)

Reply:
Got you, thanks! Fixed.